src/blockchain/Blockchain.ts

```diff
@@ -80,47 +126,26 @@ export default class Blockchain implements IBlockchainApi {
     return this.api.rpc.chain.subscribeNewHeads(listener)
   }
 
-  public async submitTx(
+  public async signTx(
```

Comment:
how about `signableTx`

Reply:
this is actually nice and descriptive: here the tx is signed